.github/workflows/build.yml

```diff
@@ -192,11 +180,32 @@ jobs:
         Add-Content -Path "$Env:GITHUB_ENV" -Value "WINDOWS_CODESIGN_CERTIFICATE_PASSWORD=$Env:WINDOWS_CODESIGN_CERTIFICATE_PASSWORD"
         Add-Content -Path "$Env:GITHUB_ENV" -Value "CMAKE_ARGS_EXTRA=-DWINDOWS_CODESIGN=ON"
 
-    - name: "Set up build environment"
-      id: buildenv
-      run: ${{ matrix.buildenv_script }} setup
-      env:
-        BUILDENV_BASEPATH: ${{ matrix.buildenv_basepath }}
+    - name: Install Nuget
+      uses: nuget/setup-nuget@v1
+      if: runner.os != 'Linux'
+      with:
+        nuget-version: 5.10.0
+
+    - name: "Authenticate to GitHub Packages"
+      if: runner.os != 'Linux' && github.repository_owner == 'mixxxdj'
+      shell: bash
+      run: |
+        nuget sources add -Name mixxx -Source https://nuget.pkg.github.com/mixxxdj/index.json -Username mixxxdj -Password ${{ secrets.GITHUB_TOKEN }} -StorePasswordInClearText -ForceEnglishOutput -NonInteractive
+        nuget setapikey ${{ secrets.GITHUB_TOKEN }} -Source mixxx -ForceEnglishOutput -NonInteractive
```

Comment:
let' use something more descriptive like "pkg-github-mixxx" or whatever describes the URL the best.